### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by mthalman** [MicrosoftDocs/vsts-rest-api-specs#499]
All the links in the content of this page are broken and just link back to the page itself.  They don't go anywhere.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6934a7ee-f487-16bc-2f06-de16f599cb16
* Version Independent ID: ee5297ca-c3e8-5c33-24dc-e4bd2c72fcea
* Content: [Overview - Azure DevOps Services REST API](https://docs.microsoft.com/en-us/rest/api/azure/devops/componentgovernance/?view=azure-devops-rest-6.0)
* Content Source: [docs-ref-conceptual/6.0/componentGovernance/index.md](https://github.com/MicrosoftDocs/vsts-docs-rest-apis/blob/live/docs-ref-conceptual/6.0/componentGovernance/index.md)
* Product: **vs-devops-alm**
* Technology: **vs-devops-integrate**
* GitHub Login: @wnjenkin
* Microsoft Alias: **whjenkin**

**Comment by kamalhabieb** [comment]
Thank you @mthalman for reporting this issue.

This issue has been fixed in the latest version https://learn.microsoft.com/en-us/rest/api/azure/devops/componentgovernance/?view=azure-devops-rest-7.1

**Comment by mthalman** [comment]
@kamalhabieb - The latest 7.1 version doesn't show any links at all:

![image](https://user-images.githubusercontent.com/15789599/191506498-3292a3a0-deeb-4e00-bada-e96577e832db.png)

**Comment by kamalhabieb** [comment]
Yes the problem of links redirecting to the same page is now fixed, but the issue that still persists is that there is no API documentation for this page.
a work item was created to the owning team, and they will be handling the fix.

Thank you for your follow up @mthalman.
